benchopt/tests/test_cli.py

```diff
@@ -348,6 +348,46 @@ class Solver(BaseSolver):
                     benchmark_dir=DUMMY_BENCHMARK_PATH
                 )
 
+    def test_handle_class_init_error(self):
+        # dataset with a wrong param name
+        dataset_src = (
+            "from benchopt import BaseDataset\n"
+            "class Dataset(BaseDataset):\n"
+            "    name = 'buggy-dataset'\n"
+            "    parameters = {'wrong_param_name': [100]}\n"
+            "    def __init__(self, param=1., deprecated_return=False):\n"
+            "        self.param = param\n"
+            "        self.deprecated_return = deprecated_return\n"
+            "    def get_data(self):\n"
+            "        X = [[1, 0], [0, 1]]\n"
+            "        y = [1, 1]\n"
+            "        return dict(X=X, y=y)\n"
```

Comment:
```suggestion
```

Reply:
It's mandatory to retain the `get_data` method as it's an abstract method of `BaseDataset`.
I kept ``return dict()``.

Reply:
perfect. thanks